package.json

```diff
@@ -85,11 +90,65 @@
     "url-loader": "^0.5.6",
     "webpack": "^1.9.6",
     "webpack-dev-middleware": "^1.2.0",
+    "webpack-dev-server": "^1.14.1",
     "webpack-hot-middleware": "^2.0.0",
     "webpack-merge": "^0.7.3"
   },
-  "devDependencies": {
-    "babel-cli": "^6.6.5",
-    "in-publish": "^2.0.0"
+  "peerDependencies": {
```

Comment:
If we want to support NPM v2, we need the dependencies installed flat so that the Karma and Webpack clis and their dependencies are accessible.

This has the nasty side-effect of installing the dependencies twice in NPM v2.